app/services/flow/steps/provider_capital_disregards/mandatory_step.rb

```diff
@@ -0,0 +1,11 @@
+module Flow
+  module Steps
+    module ProviderCapitalDisregards
+      MandatoryStep = Step.new(
+        path: ->(application) { Steps.urls.providers_legal_aid_application_means_capital_disregards_mandatory_path(application) },
+        forward: ->(application) { application.passported? ? :check_passported_answers : :check_capital_answers },
```

Comment:
Forward: I presume this is because the next page is not yet developed?

Reply:
Yes, that's right